logstash-core-event-java/src/main/java/com/logstash/Event.java

```diff
@@ -250,4 +248,8 @@ public void tag(String tag) {
             tags.add(tag);
         }
     }
+
+    public static void setLogger(Logger logger) {
+        Event.logger = (logger == null) ? DEFAULT_LOGGER : logger;
```

Comment:
Should we allow null? If DEFAULT_LOGGER is a constant, what if instead there's a setLogger(DEFAULT_LOGGER) or resetLoggerToDefault() ?